dosagelib/plugins/c.py

```diff
@@ -453,10 +453,10 @@ def namer(self, imageUrl, pageUrl):
 
 
 class CyanideAndHappiness(_BasicScraper):
-    url = 'http://www.explosm.net/'
+    url = 'http://explosm.net/comics/'
```

Comment:
```suggestion
    url = 'https://explosm.net/comics/'
```
Seems to redirect, so maybe use HTTPS directly?

Reply:
good idea, I missed that.